azkaban_cli/azkaban_cli.py

```diff
@@ -533,9 +533,32 @@ def unschedule(ctx, project, flow):
 @click.pass_context
 @click.argument(u'project', type=click.STRING)
 @click.argument(u'flow', type=click.STRING)
-def execute(ctx, project, flow):
+@click.option(u'--disabled', type=click.STRING, help=u'A list of job names that should be disabled for this execution. Should be formatted as a JSON Array String. Example Values: ["job_name_1", "job_name_2", "job_name_N"]')
+@click.option(u'--success-emails', type=click.STRING, help=u'A list of emails to be notified if the execution succeeds. All emails are delimitted with [,|;|\s+]. Example Values: foo@email.com,bar@email.com')
+@click.option(u'--failure-emails', type=click.STRING, help=u'A list of emails to be notified if the execution fails. All emails are delimitted with [,|;|\s+]. Example Values: foo@email.com,bar@email.com')
+@click.option(u'--success-emails-override/--no-success-emails-override', help=u'Whether uses system default email settings to override successEmails.')
```

Comment:
Is this a boolean or a string? It can be `true/false` according to documentation, but I'm not sure if Python booleans get converted automatically. In any case, since there is no explicit type definition, it will be a string for Click. Will passing `"true"` or `"false"` work just as well?
It's my only point in the PR. Did you happen to try it out to see what happens? It's such a niche feature that I don't think it blocks merging the PR, but I'm mostly just curious to see how it works ;)

Reply:
@aguimaraesduarte in case you're talking about `--success-emails-override` or `--no-success-emails-override` flags, they are treated like boolean values. There are no type definition, but when click sees a slash in option string, it will assume that is a boolean flag. The default behaviour of this flags is to be `False`, as is when you pass the flag that begins with `--no`.

Everything I said here is on that part of click docs:
http://click.palletsprojects.com/en/5.x/options/#boolean-flags

Hope it helps to understand. 👍

Reply:
Cool! Didn't know that. Awesome